### ISSUE TITLE
NTR: Cardiolipin ratio abnormal in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8368]
New term request Cardiolipin ratio abnormal in blood (elthat_131030201145): MLCL:L4-CL; monolysocardiolipins:tetralinoleoyl cardiolipin

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000517
- label: Elevated monolysocardiolipin/cardiolipin ratio
- parents: [Abnormal circulating phospholipid concentration (HP:0040176)](https://hpo.jax.org/app/browse/term/HP:0040176)
- pmid: PMID:34382226
- def: The concentration of monolysocardiolipin related to that of cardiolipin is above the upper limit of normal. This is a functional assay that can be performed from a blood spot to diagnose Barth syndrome (OMIM:302060).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8368
- synonyms:
